dashboards/resources.libsonnet

```diff
@@ -172,6 +172,211 @@ local g = import 'grafana-builder/grafana.libsonnet';
         )
       ),
 
+    'k8s-resources-workloads-namespace.json':
+      local tableStyles = {
+        workload: {
+          alias: 'Workload',
+          link: '%(prefix)s/d/%(uid)s/k8s-resources-workload?var-datasource=$datasource&var-cluster=$cluster&var-namespace=$namespace&var-workload=$__cell' % { prefix: $._config.grafanaPrefix, uid: std.md5('k8s-resources-workload.json') },
+        },
+        workload_type: {
+          alias: 'Workload Type',
+        },
+      };
+
+      local cpuUsageQuery = |||
```

Comment:
This is clean style, with the queries defined in locals, I like it. It's a little different style than other grafana-builder dashboards, but looks clean.